packages/ethereum-storage/test/gas-price-definer.test.ts

```diff
@@ -110,12 +154,17 @@ describe('GasPriceDefiner', () => {
       ]);
     });
 
-    it('returns empty array if there is no provider', async () => {
+    it('returns  empty array if there is no provider for the ethereum ', async () => {
```

Comment:
What means `for the ethereum`  ?

Reply:
API endpoints for fetching price for  the type of transactions provided in `StorageType.IGasPriceProvider`  , but i  got the point that it sounds confusing .